tokio-trace-futures/examples/proxy_server.rs

```diff
@@ -23,10 +23,14 @@ use tokio::net::{TcpListener, TcpStream};
 use tokio::prelude::*;
 
 fn main() -> Result<(), Box<std::error::Error>> {
-    let listen_addr = env::args().nth(1).unwrap_or("127.0.0.1:8081".to_string());
+    let listen_addr = env::args()
+        .nth(1)
+        .unwrap_or_else(|| "127.0.0.1:8081".to_string());
     let listen_addr = listen_addr.parse::<SocketAddr>()?;
 
-    let server_addr = env::args().nth(2).unwrap_or("127.0.0.1:3000".to_string());
+    let server_addr = env::args()
+        .nth(2)
+        .unwrap_or_else(|| "127.0.0.1:3000".to_string());
```

Comment:
Out of curiosity: is this a rustfmt suggestion or a clippy suggestion?

Reply:
looks like rustfmt, I have it setup to run on save so its pretty easy for it to sneak in reformats :S

Reply:
I think this is a clippy one so that you only run to_string one the unwrap on an err iirc.

Reply:
That's exactly what it was, good catch.

Reply:
Oh I missed that, thanks.

FWIW the formatting changes are fine regardless, I was just curious.